fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -3423,7 +3423,7 @@ ACTOR Future<TLogLockResult> TagPartitionedLogSystem::lockTLog(
 			                               : Never())) {
 				TraceEvent("TLogLocked", myID).detail("TLog", tlog->get().id()).detail("End", data.end);
 				if (lockInterf.present()) {
-					lockInterf.get()->lockInterf[tlog->get().id()] = tlog->get().interf();
+					lockInterf.get()->lockInterf[data.id] = tlog->get().interf();
 				}
```

Comment:
Why is the "id" changing? Do we understand that?

Reply:
Sorry, I do not understand your question. But I will explain what the PR is doing. We are populating a map. The key to the map is an id representing the tLog. Prior to the PR I used the wrong key. I used the id associated with the interface. This is not the same id that is in the TLog Lock reply message. 

The id that is in the lock reply is the dbgid. Later in the recovery, we enumerate every lock reply, and for each, look up in the map with key = the dbgid in the lock reply. Without the PR, we never found a match in the map, because we populated the map using the interface id, rather than the dbgid.

Reply:
Thanks for the explanation!